src/components/Calculator/index.tsx

```diff
@@ -0,0 +1,674 @@
+  )
+}
+const RightSideCalculator = ({
+  totalTokens,
+  setTotalTokens,
+}: {
+  totalTokens: number
+  setTotalTokens: Dispatch<SetStateAction<number>>
+}) => {
+  const { t } = useTranslation()
+
+  const totalPrice = TOKEN_PRICE * totalTokens
+
+  return (
+    <Flex
+      flexDirection='column'
+      flexGrow={1}
+      flexShrink={0}
+      flexBasis={{ base: '100%', xl: '50%' }}
+      bgColor='calculator.right_side'
+      color='calculator.right_side_color'
+    >
+      <TabPanels flexGrow={1} display='flex' flexDirection='column' pt={10}>
+        <TabPanel flexGrow={1} display='flex' flexDirection='column'>
+          <Box px={{ base: 5, lg: 14 }}>
+            <Flex justifyContent='space-between' mb={3}>
+              <Text>{t('calculator.voting_process_cost')}</Text>
+              <Text fontWeight='bold'>{totalTokens} tokens</Text>
+            </Flex>
+            <Flex justifyContent='space-between' mb={8}>
+              <Text>{t('calculator.price_per_token')}</Text>
+              <Text fontWeight='bold'>0.15 €</Text>
+            </Flex>
+            <Flex justifyContent='space-between'>
+              <Flex flexDirection='column'>
+                <Text>{t('calculator.total_cost')}</Text>
+                <Text>{t('calculator.vat')}</Text>
+              </Flex>
+              <Text fontWeight='bold'>{totalPrice.toFixed(2)} €</Text>
+            </Flex>
+          </Box>
+          <Box mt='auto'>
+            <BuyBtns totalTokens={totalTokens} />
+
+            <Text fontSize='14px' textAlign='center'>
+              <Trans
+                i18nKey='calculator.contact'
+                components={{
+                  customLink: (
+                    <Link as={ReactRouterLink} to='https://www.vocdoni.io/contact' target='_blank' color='white' />
+                  ),
+                  faqsLink: <Link as={ReactRouterLink} to='/#faqs' color='white' />,
+                }}
+              />
+            </Text>
+          </Box>
+        </TabPanel>
+
+        <TabPanel flexGrow={1} display='flex' flexDirection='column'>
+          <Flex mb={3} px={{ base: 5, lg: 14 }}>
+            <Text flex='1 1 33%' textAlign='start' textTransform='uppercase'>
+              {t('calculator.package')}
+            </Text>
+            <Text flex='1 1 30%' textAlign='center' textTransform='uppercase'>
+              {t('calculator.discount')}
+            </Text>
+            <Text flex='1 1 30%' textAlign='end' textTransform='uppercase'>
+              {t('calculator.total')}
+            </Text>
+          </Flex>
+          <RadioGroup
+            value={String(totalTokens)}
+            onChange={(e) => {
+              if (e === '1000') setTotalTokens(1000)
+              else if (e === '15000') setTotalTokens(15000)
+              else if (e === '30000') setTotalTokens(30000)
+              else if (e === '50000') setTotalTokens(50000)
+              else if (e === '100000') setTotalTokens(100000)
+            }}
```

Comment:
Wut? Why not simply...
```suggestion
            onChange={(e) => {
              setTotalTokens(Number(e))
            }}
```